hydra-explorer/src/Hydra/Explorer/Options.hs

```diff
@@ -0,0 +1,41 @@
+module Hydra.Explorer.Options where
+
+import Hydra.Prelude
+
+import Hydra.Cardano.Api (ChainPoint (..), NetworkId, SocketPath)
+import Hydra.Network (Host)
+import Hydra.Options (
+  networkIdParser,
+  nodeSocketParser,
+  peerParser,
+  startChainFromParser,
+ )
+import Options.Applicative (Parser, ParserInfo, fullDesc, header, helper, info, progDesc)
+
+type Options :: Type
+data Options = Options
+  { networkId :: NetworkId
+  , host :: Host
+  , nodeSocket :: SocketPath
+  , startChainFrom :: Maybe ChainPoint
+  }
+  deriving stock (Show, Eq)
+
+optionsParser :: Parser Options
+optionsParser =
+  Options
+    <$> networkIdParser
+    <*> peerParser
```

Comment:
Must: not use `--peer` to set the listen port of a web server.

The help text is irrelevant and this is confusing.

I think we should have a `--listen <host>:<port>` (or `--api-host` and `--api-port` like the hydra-node if you want to re-use things) or only a `--port <port>`.

Further, I think re-using the parsers (including help text and flags) from `hydra-node` is less clear than just duplicating the code to here.